src/components/common/MediaItem.tsx

```diff
@@ -15,6 +15,7 @@ export type MediaFile = {
   caption?: string;
   type: 'Photo' | 'Video';
   position?: ObservationMonitoringPlotPosition;
+  quadrat: boolean;
```

Comment:
Should probably prefix this with `is` to improve clarity.

```suggestion
  isQuadrat: boolean;
```